actors/management/commands/import_best_actors.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+
+from actors.get_actors import actor, import_data
+
+
+class Command(BaseCommand):
+    help = 'Import AffiliatorCommitters data'
+
+    ACTORS = staticmethod(actor)
+    IMPORT_DATA = staticmethod(import_data)
+
+    def handle(self, *args, **options):
+    	i = 1
+    	for actor in self.ACTORS:
+        	self.IMPORT_DATA(i, actor)
+        	i = i + 1
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp3p5cz0y5/actors/management/commands/import_best_actors.py
+++ b/tmp/tmp3p5cz0y5/actors/management/commands/import_best_actors.py
@@ -13,4 +13,4 @@
     	i = 1
     	for actor in self.ACTORS:
         	self.IMPORT_DATA(i, actor)
-        	i = i + 1
+            i = i + 1
```